source/components/home/navigation/navigation.js

```diff
@@ -37,6 +37,8 @@ class Navigation extends Component {
                         <NavItem onClick={() => this.scrollTo('Speakers')} eventKey={2} >Past Speakers</NavItem>
                         <NavItem onClick={() => this.scrollTo('Sponsors')} eventKey={3} >Past Sponsors</NavItem>
                         <NavItem onClick={() => this.scrollTo('Footer')} eventKey={4} >Contact Us</NavItem>
+                        <NavItem onClick={() => location.href="https://blog.reflectionsprojections.org"} eventKey={5}>Blog</NavItem>
```

Comment:
Can you swap the order a little and make Contact Us the last menu item? Blog -> FAQ -> Contact Us